Tests/FluentAssertions.Equivalency.Specs/SelectionRulesSpecs.cs

```diff
@@ -1779,10 +1779,33 @@ public void When_respecting_declared_types_explicit_interface_member_on_interfac
             subject.VehicleId = 1; // interface member
 
             // Act
-            Action action = () => subject.Should().BeEquivalentTo(expected, opt => opt.RespectingDeclaredTypes());
+            Action action = () => subject.Should().BeEquivalentTo(expected);
 
             // Assert
-            action.Should().NotThrow();
+            action.Should().Throw<XunitException>();
```

Comment:
Because of the changed/improved behavior, the _normal_ property `VehicleId` will now be taking into account instead of the one that was explicitly implemented. And since the _normal_ `VehicleId` of the subject is 2, and `1` is expected, it _should_ throw.